lab-bundle-spec.md

```diff
@@ -199,10 +207,12 @@ fleet                         |          | enum*   | Specify a Qwiklabs fleet to
 attribute   | required | type       | notes
 ----------- | -------- | ---------- | ----------------------------------------
 permissions |          | array      | Array of project/roles(array) pairs
+ssh_project |          | string     | If an SSH key is desired for thise user, the id of the project that the SSH key should be attached to.
```

Comment:
I don't understand the SSH stuff that well, so this may or may not make sense. Looking for feedback on this.

Reply:
Navy was recently updated such that an SSH key is now created for each gaia user and the related information is now stored on the website side, so the user flag here is probably no longer necessary.
On the other side, the project needs to specify which user's SSH key should be associated with the project

Reply:
Do we actually need to create an ssh key for each user though? Or could we just do it when specified?
As for the project side, I figured if we had the user side indicate which project to attach to, then we could extract that into the project details at unbundling time by finding the project with the id in question.

Reply:
Navy currently creates an SSH key for each user, so there's no additional overhead on the website side. this was done in order to cut down on the number of GCP API calls needed during lab launch 
Multiple projects could use (and if they use any, it's likely they will use) the same default SSH key

Reply:
Ah gotcha. Okay - updated in https://github.com/CloudVLab/qwiklabs-content-bundle-spec/pull/34/commits/e8a574e90992f14d50cb7fb5cfbda423ddb92e86.